src/settings/Addresses.js

```diff
@@ -115,7 +115,7 @@ class Addresses extends Component {
   onCreate = item => ({
     value: JSON.stringify(item),
     module: moduleName,
-    configName: `${moduleName.toLowerCase()}.${configName}`,
```

Comment:
but `${moduleName.toLowerCase()}.${configName}` === `tenant.addresses`, no?

Reply:
`${moduleName.toLowerCase()}.${configName}` === `tenant.tenant.addresses`

Reply:
IMHO, explicit `const configName = 'tenant.addresses';` is better

Reply:
I mean with previous code `${moduleName.toLowerCase()}.${configName}` is `tenant.addresses`